qutebrowser/browser/downloads.py

```diff
@@ -296,10 +296,10 @@ def _create_fileobj(self):
         else:
             self.set_fileobj(fileobj)
 
-    def _ask_overwrite_question(self):
+    def _ask_confirm_question(self, msg):
         """Create a Question object to be asked."""
         q = usertypes.Question(self)
-        q.text = self._filename + " already exists. Overwrite? (y/n)"
+        q.text = msg + ' (N/y)'
```

Comment:
Actually I don't think the `' (N/y)'` should be added here (even if it was before) - this should probably be done in `prompter.py` instead, and just display `(no)`. Are you okay with removing this?

(I know the `(yes)`/`(no)` notation is slightly weird, but `y`/`n` might not be the right keys if the user remapped them, so that might be confusing as well)